scripts/copyResearchDocs.ts

```diff
@@ -20,6 +23,17 @@ import { adapters, type Tracker } from 'trackhar';
         '--json',
     ]).then(({ stdout }) => JSON.parse(stdout).data.trees[0].name.split('@')[1] as string);
 
+    if (!process.env['FORCE_REGENERATE'] && (await stat(trackharVersionFile))) {
```

Comment:
I definitely see the need for this optimisation in dev, but I am not certain about CI. But I guess we can test whether it's fine like this and otherwise set the env var in CI later.